Dockerfile

```diff
@@ -32,16 +32,16 @@ RUN \
         yum-config-manager --enable osg-${BASE_YUM_REPO}; \
     fi && \
     yum clean all && \
-    rm -rf /var/cache/yum/
+    rm -rf /var/cache/yum/ && \
+    # Impatiently ignore the Yum mirrors
+    sed -i 's/\#baseurl/baseurl/; s/mirrorlist/\#mirrorlist/' \
+        /etc/yum.repos.d/osg{,-upcoming}-testing.repo && \
+    mkdir -p /etc/osg/image-{cleanup,init}.d/ && \
+    # Support old init script dir name
+    ln -s /etc/osg/image-{init,config}.d
 
-# Impatiently ignore the Yum mirrors
-RUN sed -i 's/\#baseurl/baseurl/; s/mirrorlist/\#mirrorlist/' \
-        /etc/yum.repos.d/osg{,-upcoming}-testing.repo
-
-RUN mkdir -p /etc/osg/image-config.d/
-ADD image-config.d/* /etc/osg/image-config.d/
```

Comment:
do these `image-config.d/*` files still need to be copied in?

Reply:
Nope, I deleted them in another commit. They were all empty files anyway